javascript/javascript1/week3/seriesDurationOfMyLife.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Javascript file for Hack Your Future, Javascript1, homework Week3
+ * program will calculate how much time a tv series have taken as a 
+ * percentage of an average lifespan of 80 years.
+*/
+
+// Series duration of my life
+
+const seriesDurations = [
+    {
+        title: "Breaking Bad",
+        days: 5,
+        hours: 10,
+        minutes: 12,
+    },
+    {
+        title: "Prison Break",
+        days: 8,
+        hours: 4,
+        minutes: 14,
+    },
+    {
+        title: "See",
+        days: 4,
+        hours: 12,
+        minutes: 11,
+    },
+];
+
+const lifeYearsInMinutes = 80 * 525600;
```

Comment:
The number `525600` is not obvious what it represents. I guess it is minutes in a year. To help the next person who will read your code give it a name. The next coder has to figure out what your code means so help them out with some clues.
```suggestion
const minutesInYear = 525600;
const lifeYearsInMinutes = 80 * minutesInYear;
```

Reply:
Thanks for the comment. That's true.